lib/oauth2/client.rb

```diff
@@ -156,9 +158,8 @@ def request(verb, url, opts = {})
     # @param params [Hash] a Hash of params for the token endpoint
     # @param access_token_opts [Hash] access token options, to pass to the AccessToken object
     # @param extract_access_token [Proc] proc that extracts the access token from the response (DEPRECATED)
-    # @param access_token_class [Class] class of access token for easier subclassing OAuth2::AccessToken, @version 2.0+
     # @return [AccessToken] the initialized AccessToken
-    def get_token(params, access_token_opts = {}, extract_access_token = options[:extract_access_token], access_token_class: AccessToken)
+    def get_token(params, access_token_opts = {}, extract_access_token = options[:extract_access_token])
```

Comment:
@stanhu Sorry I didn't notice this earlier, but this doesn't appear to be functional.  There wasn't a test that covered it, so I can't be sure.  I refactored it to move the reference to `options` inside the method, and added specs.  Fix will be in 2.0.3 out soon.

Reply:
@pboling Thanks. What was the issue? I thought this was the same signature used in v1.4.9.